manifests/params.pp

```diff
@@ -2,8 +2,14 @@
 class openldap::params {
   case $::osfamily {
     'Debian': {
-      $client_package           = 'libldap-2.4-2'
-      $client_conffile          = '/etc/ldap/ldap.conf'
+      $client_package           = $::osfamily ? {
```

Comment:
really not sure why i ended up doing that. and was just looking at this file w/o the context of your comments and came to the same conclusion.